scripts/apps/search/services/TagService.js

```diff
@@ -79,7 +80,7 @@ export function TagService($location, desks, userList, metadata, search,
 
             if (!added) {
                 var paramArr = parameter.split(':');
-                var parameterTranslated = gettextCatalog.getString(paramArr[0]) + ':' + paramArr[1];
+                var parameterTranslated = gettext(paramArr[0]) + ':' + paramArr[1];
```

Comment:
Do not use `gettext` for dynamic values. Instead an alias of `gettext` should be used so the code inside `gettext` isn't extracted and sent to Transifex.

It doesn't look like it makes sense to translate this at all.